.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -14,8 +14,7 @@
 ## Checklist if releasing new version:
 - [ ] I made sure that both API and Admin changes are present in Notify
 - [ ] I have checked if the docker images I am referencing exist - ex: https://gcr.io/cdssnc/notify/admin:7ddcb76
-- [ ] I have made the #team-sre team aware that I am pushing changes
 
 ## Checklist if making changes to Kubernetes:
 - [ ] I have made #team-sre team aware I am pushing changes
```

Comment:
We can probably remove it from this section as well

Reply:
Just removed!